tests/operators/test_latest_only_operator.py

```diff
@@ -19,7 +19,6 @@
 
 import datetime
 import unittest
-
```

Comment:
Could you revert this? It's the only change in this file, and I like a blank line between stdlib imports and third-party ones personally.

Reply:
Sure. I did not notice :).